test/functional/apps/visualize/_heatmap_chart.js

```diff
@@ -114,22 +114,12 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.visualize.clickOptionsTab();
       await PageObjects.visualize.clickEnableCustomRanges();
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.isCustomRangeTableShown();
-      await PageObjects.visualize.addCustomRange(0, 100);
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.addCustomRange(100, 200);
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.addCustomRange(200, 300);
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.addCustomRange(300, 400);
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.addCustomRange(400, 500);
       await PageObjects.visualize.clickAddRange();
-      await PageObjects.visualize.addCustomRange(500, 600);
       await PageObjects.visualize.clickAddRange();
```

Comment:
Maybe we could change one of the default ranges just to test whether values are propagated correctly in this case.

Reply:
This makes sense!
I've added a couple of custom ranges here.
@dmlemeshko could you please take a look ?

Reply:
I like the idea to check custom range input, can we keep the old function then?

```
async addCustomRange(index, from, to) {
      await testSubjects.setValue(`heatmapColorRange${index}__from`, 'from);
      await testSubjects.setValue('heatmapColorRange${index}__to', to);
}
```

Reply:
@dmlemeshko done.
I just named it as `setCustomRangeByIndex`, because it actually doesn't create new ranges, but customizes the existing by index.